src/librustc/diagnostics.rs

```diff
@@ -112,6 +112,11 @@ reference when using guards or refactor the entire expression, perhaps by
 putting the condition inside the body of the arm.
 "##,
 
+E0018: r##"
+The value of static and const variables must be known at the compile-time which is
+not possible with a pointer or with a value pointed by a pointer.
```

Comment:
"value pointed by a pointer" -- I'm not sure what to change this to, but it's not exactly grammatically correct as is.